src/components/v2/shared/DistributionSplitsSection/SpecificLimitModal.tsx

```diff
@@ -25,7 +25,11 @@ export default function SpecificLimitModal({
   function setNewSplitsFromLimit() {
     form.validateFields()
 
-    setDistributionLimit(form.getFieldValue('distributionLimit'))
+    const distributionLimit = form
```

Comment:
@wraeth-eth i think we should fix this at a lower level. Ideally the `distributionLimit` value never contains commas. I think this is how other things work throughout the app? (no 100% sure)

Reply:
definitely agree - I have some stuff in the pipeline actually to fix this (I hope). I think we should still get this one out to prod asap, wdyt?

FWIW - currently reworking the form stuff to be more in line with what antd likes

Reply:
yeah k